### PR TITLE
Add skeleton loading state

### PR DESCRIPTION
**Pull request by kissmikijr**
#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [ ] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: plugins/frontend/backstage-plugin-shortcut/src/components/Home/StoriesCardHomepage.tsx [hunk 1]
```diff
@@ -195,7 +237,12 @@ const StoriesCardContent = () => {
   });
 
   if (loading) {
-    return <Progress />;
+    return (
+      <>
+        <Skeleton variant="text" width={250} />
```
Comment (Irma12) [inline plugins/frontend/backstage-plugin-shortcut/src/components/Home/StoriesCardHomepage.tsx:242]: How does it behave on mobile screens?

File: plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsListView/PullRequestsListView.tsx [hunk 2]
```diff
@@ -66,6 +67,45 @@ const useStyles = makeStyles(theme => ({
...
+        spacing={1}
+        xs={1}
+        style={{ flexShrink: 0, marginLeft: 'auto' }}
+        className={classes.secondaryText}
+      >
+        <Box display="flex" justifyContent="flex-end">
+          <Skeleton
+            variant="circle"
+            width={18}
+            height={18}
```
Comment (Irma12) [inline plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsListView/PullRequestsListView.tsx:101]: How do you determine size which is optimal?

File: plugins/frontend/backstage-plugin-shortcut/src/components/Home/StoriesCardHomepage.tsx [hunk 3]
```diff
@@ -179,9 +220,10 @@ const StoriesCardContent = () => {
   const { value, loading, error } = useAsync(async () => {
     const profile = await identityApi.getProfileInfo();
     const allMembers = await api.getUsers();
-    const loggedUser = allMembers?.find(
-      user => user.profile.email_address === profile.email,
-    )?.profile.mention_name;
+    // const loggedUser = allMembers?.find(
+    //   user => user.profile.email_address === profile.email,
+    // )?.profile.mention_name;
+    const loggedUser = 'mikloskiss';
```
[the remark this answers is not in the document]
Comment (kissmikijr) [reply plugins/frontend/backstage-plugin-shortcut/src/components/Home/StoriesCardHomepage.tsx:226]: yea , haha, thanks!

File: plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsListView/PullRequestsListView.tsx (hunk 2 above)
Comment (kissmikijr) [reply plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsListView/PullRequestsListView.tsx:101]: trial and error :D I think it looks ok with the 18px

**Review by Irma12 — APPROVED** [review]

**Pull request merged**